src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -635,6 +636,7 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
 
         private async Task OnDocumentChangeProcessingCompleteAsync(IDocument doc, VersionCookie vc, bool enqueueForAnalysis, AnalysisPriority priority, IDisposable disposeWhenEnqueued) {
             try {
+                _shutdownCts.Token.ThrowIfCancellationRequested();
```

Comment:
Then we don't need to call `_disposableBag.ThrowIfDisposed();` here (and catch it at the end).

Reply:
Actually OnDocumentChangeProcessingCompleteAsync is not even async anymore